scripts/encrypt_conf.py

```diff
@@ -7,13 +7,28 @@
    scripts/encrypt_conf.py confname1 confname2 ... confnameN
    scripts/encrypt_conf.py credentials
 """
-import sys
-
+import argparse
 import yaycl_crypt
 
 from utils import conf
 
-for conf_name in sys.argv[1:]:
-    conf_name = conf_name.strip()
+
+def parse_cmd_line():
+    parser = argparse.ArgumentParser(argument_default=None)
+    parser.add_argument('-e', '--encrypt', default=False, dest='encrypt', action='store_true',
+                        help='encrypts the file specified')
+    parser.add_argument('-d', '--decrypt', default=False, dest='decrypt', action='store_true',
+                        help='decrypts the file specified')
+    parser.add_argument('--file', dest='file', default='credentials',
+                        help='file name in "conf" to be encrypted/decrypted')
+    args = parser.parse_args()
+    return args
+
+args = parse_cmd_line()
+conf_name = args.file.strip()
+if args.encrypt:
     yaycl_crypt.encrypt_yaml(conf, conf_name)
```

Comment:
can you add kwarg `delete=False` to prevent it from deleting files? This _just_ bit and I lost changes because of it.